compiler/test/dotty/tools/dotc/FromTastyTests.scala

```diff
@@ -89,6 +89,7 @@ class FromTastyTests extends ParallelTesting {
          "t493.scala",
          "t8395.scala",
          "t3613.scala",
+         "quote-no-splices.scala",
```

Comment:
I will fix this issue in another PR, it is unrelated to the`Expr.{run|show}` implementation.